src/main.rs

```diff
@@ -162,12 +162,12 @@ enum Commands {
         basin: S2BasinUri,
 
         /// Create stream on append with basin defaults if it doesn't exist.
-        #[arg(long)]
-        create_stream_on_append: Option<bool>,
+        #[arg(long, default_value_t = false)]
+        create_stream_on_append: bool,
 
         /// Create stream on read with basin defaults if it doesn't exist.
-        #[arg(long)]
-        create_stream_on_read: Option<bool>,
+        #[arg(long, default_value_t = false)]
+        create_stream_on_read: bool,
```

Comment:
wouldn't this cause spurious reconfiguration on basins when not explicitly specified?